### PR TITLE
Fix problems for low resolution desktop devices

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
@DevExpress/testcafe-docs 

Test at http://strelyaev-w8:8080/testcafe/documentation/test-api/test-code-structure.html at the resolution of 1366x768

### CODE REVIEW

File: src/_includes/head.html [hunk 1]
```diff
@@ -61,14 +61,19 @@
   {% if isDocumentation %}
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jquery.perfect-scrollbar/0.6.12/css/perfect-scrollbar.min.css">
     <script defer type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery.perfect-scrollbar/0.6.12/js/perfect-scrollbar.min.js"></script>
+    <script defer type="text/javascript" src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-2.2.3.min.js"></script>
+    <script defer src="{{ "/scripts/widget.js" | prepend: site.baseurl }}" type="text/javascript"></script>
```
Comment (VasilyStrelyaev) [inline src/_includes/head.html]: These lines were already there - don't know why they were included in this commit

Comment (kirovboris) [inline src/_includes/head.html]: Did you forget to make rebase?

File: src/_includes/head.html [hunk 2]
```diff
@@ -61,14 +61,19 @@
...
+
     <script>
         var MIN_BODY_WIDTH = 1000;
         var FOOTER_HEIGHT  = 63;
         var HEADER_HEIGHT  = 102;
 
         var nav            = null;
         var article        = null;
-        var isMobileDevice = window.screen.width < 800 || window.screen.height < 800;
+        var md = new MobileDetect(window.navigator.userAgent);
```
Comment (kirovboris) [inline src/_includes/head.html]: Please fix indentation

File: src/_includes/head.html (hunk 1 above)
Comment (VasilyStrelyaev) [inline src/_includes/head.html]: I didn't, it's up-to-date

Comment (AlexanderMoskovkin) [inline src/_includes/head.html]: You've added these lines in the PR with the widget. It's not merged yet

**Comment by AlexanderMoskovkin** [comment]
I've merged a PR with the widget. Please rebase this PR now

**Comment by VasilyStrelyaev** [comment]
FPR

**Pull request merged**